### PR TITLE
coding blocks added

### LINKED ISSUE #1566: [Add] Coding Blocks IDE
### Site URL

https://ide.codingblocks.com/

### Description of the above provided site

In addition to the IDE with code completion, you can also create an account to save your code on the server, and also share it with others.

### Category

Frontend

### Sub Category

code editor

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by S-ishita**
## Fixes Issue
closes #1566 

## Screenshots
Screenshot from 2023-08-10 23-17-37

### CODE REVIEW

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -117,5 +117,12 @@
     "url": "https://ide.geeksforgeeks.org/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+    "name": "Coding Blocks",
+    "description": "In addition to the IDE with code completion, you can also create an account to save your code on the server, and also share it with others.",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:123]:
```suggestion
    "description": "In addition to the IDE with code completion, you can also create an account to save your code on the server and share it with others.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just made a tweak @S-ishita

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by rupali-codes** [comment]
@S-ishita could you please resolve merge conflicts?

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

> I have resolved the merge conflicts.

**Comment by Anmol-Baranwal** [comment]
@CBID2 
Review and merge this.

Let's clean up the issues and PRs.

**Comment by CBID2** [comment]
> @CBID2 
> 
> Review and merge this.
> 
> 
> 
> Let's clean up the issues and PRs.

Yup

**Review by CBID2 — APPROVED** [review]
LGTM

**Pull request merged**